src/Model/ShippingMethod.php

```diff
@@ -11,11 +11,14 @@ public static function fromData(array $data): self
             $prices[$country['iso_2']] = (int)($country['price'] * 100);
         }
 
+        $min_weight = intval(ceil($data['min_weight'] * 1000));
+        $max_weight = intval(ceil($data['max_weight'] * 1000));
+
         return new self(
             (int)$data['id'],
             (string)$data['name'],
-            (int)($data['min_weight'] * 1000),
-            (int)($data['max_weight'] * 1000),
+            (int) $min_weight,
+            (int) $max_weight,
```

Comment:
Isn't it better to use `round()` for this as it corrects float calculation imprecision both ways?:

```suggestion
            (int)round($data['min_weight'] * 1000.0),
            (int)round($data['max_weight'] * 1000.0),
```

This also inlines the operation again. If there are no additional steps involved I think introducing a new variable reduces readability.